amf-api-contract/shared/src/main/scala/amf/plugins/domain/shapes/validation/JsonCandidateValidator.scala

```diff
@@ -0,0 +1,32 @@
+package amf.plugins.domain.shapes.validation
+
+import amf.ProfileName
+import amf.client.remod.amfcore.plugins.validate.ValidationConfiguration
+import amf.core.validation._
+import org.mulesoft.common.core.CachedFunction
+import org.mulesoft.common.functional.MonadInstances.identityMonad
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object JsonCandidateValidator {
```

Comment:
Found this name confusing as this object is simple facade which is agnostic to specific syntax.

Reply:
Seemed appropiate to me as this validates candidates with "application/json" mediatype

Reply:
changed it for CandidateValidator